tools/scripts-config-eslint/README.md

```diff
@@ -7,7 +7,7 @@ To use the eslint configuration in your IDE, or to add custom rules
 
 ```json
 {
-	"extends": "./node_modules/@talend/scripts-config-eslint/.eslintrc.js",
+	"extends": "@talend/scripts-config-eslint",
```

Comment:
Linked to previous comment in package.json.
If the change is confirmed, we need to update this file